src/components/dialogmote/avlys/AvlysDialogmoteBegrunnelse.tsx

```diff
@@ -17,27 +18,36 @@ interface AvlysDialogmoteBegrunnelseProps {
 const AvlysDialogmoteBegrunnelse = ({
   fieldName,
   label,
-}: AvlysDialogmoteBegrunnelseProps): ReactElement => (
-  <AvlysDialogmoteSkjemaSeksjon>
-    <FlexRow bottomPadding={PaddingSize.SM}>
-      <FlexColumn flex={1}>
-        <Field<string> name={fieldName}>
-          {({ input, meta }) => (
-            <FritekstStor
-              label={label}
-              maxLength={1000}
-              feil={meta.submitFailed && meta.error}
-              id={fieldName}
-              {...input}
-            />
-          )}
-        </Field>
-      </FlexColumn>
-    </FlexRow>
-    <FlexRow>
-      <Knapp htmlType="button">{texts.preview}</Knapp>
-    </FlexRow>
-  </AvlysDialogmoteSkjemaSeksjon>
-);
+}: AvlysDialogmoteBegrunnelseProps): ReactElement => {
+  const trackButtonClick = useTrackButtonClick();
```

Comment:
Kanskje man kunne laget en `<TrackedButton>`-komponent som kan wrappe en hvilken som helst button og bruke hooken til å tracke klikk på buttons den wrapper. Har ikke helt klart for meg hvordan det vil funke, men noe å vurdere til videre PRer 😉

Reply:
Hm ja, det kunne nok gått an. Skal se om jeg får det til.